src/App.js

```diff
@@ -14,7 +14,6 @@ const App = () => (
     <div>
```

Comment:
- [ ] The App is currently not working as the functionalities are yet to be implemented correctly, kindly modify them.
- [ ] Kindly make the README file more descriptive.